### PR TITLE
Feature/pnp 9 meta social

### PR DESCRIPTION
**Pull request by igor-kamil**
https://jira.sng.sk/browse/PNP-9

### CODE REVIEW

File: public/safari-pinned-tab.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="808.000000pt" height="808.000000pt" viewBox="0 0 808.000000 808.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.11, written by Peter Selinger 2001-2013
+</metadata>
+<g transform="translate(0.000000,808.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+</g>
+</svg>
```
Comment (rastislav-chynoransky) [inline public/safari-pinned-tab.svg:13]: Is this empty svg on purpose?

File: resources/views/components/favicons.blade.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/site.webmanifest">
+<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#c00c38">
+<meta name="msapplication-TileColor" content="#da532c">
```
Comment (rastislav-chynoransky) [inline resources/views/components/favicons.blade.php:6]: Orange?

**Review by rastislav-chynoransky — APPROVED** [review]

File: resources/views/components/favicons.blade.php (hunk 2 above)
Comment (igor-kamil) [reply resources/views/components/favicons.blade.php:6]: msapplication 💁
but fixed

File: public/safari-pinned-tab.svg (hunk 1 above)
Comment (igor-kamil) [reply public/safari-pinned-tab.svg:13]: I use https://realfavicongenerator.net/ for generating. and it comes within the bundle - actually - it's not empty. it's EXL for touchbar on macbooks...

Comment (rastislav-chynoransky) [reply public/safari-pinned-tab.svg:13]: There is just one translated `<g>` :woman_shrugging:  but it really doesn't matter...

**Pull request merged**